tools/bin/check-duplicated-storages.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
We should use `#!/usr/bin/env bash` for [portability](https://en.wikipedia.org/wiki/Shebang_(Unix)#Portability)
```suggestion
#!/usr/bin/env bash
```